data/hai/hai reveal 2 start.txt

```diff
@@ -51,7 +51,7 @@
 mission "Hai Rescue: Yeertle Hint"
 	landing
 	name "Investigate a Were-Squirrel"
-	description "A truly outlandish rumor has been floating around that there might be a "were-squirrel" (Hai) hiding out on <destination>."
+	description "A truly outlandish rumor has been floating around that there might be a “were-squirrel” (Hai) hiding out on <destination>."
```

Comment:
I believe this is how it actually needs to be done because fancy quotes aren't supported.
```suggestion
	description `A truly outlandish rumor has been floating around that there might be a "were-squirrel" (Hai) hiding out on <destination>.`
```